src/render.js

```diff
@@ -5,34 +5,29 @@ const { remote, ipcRenderer } = require('electron');
 const Store = require('electron-store');
 const Generator = require('cucumber-forge-report-generator');
 
-const DarkMode = require('darkmode-js');
-
-let darkMode = null;
-let darkModeCheckBox = null;
-
 const store = new Store();
 const fileEncoding = 'utf-8';
 
+let darkModeCheckBox;
 let selectedFolderPath;
 let projectName;
 let reportHTML;
 let tag;
 
 const initDarkMode = () => {
   darkModeCheckBox = document.getElementById('darkMode');
-  if (darkModeCheckBox) {
-    darkModeCheckBox.addEventListener('click', () => {
-      if (!darkMode) {
-        darkMode = new DarkMode();
-        darkMode.toggle();
-      } else {
-        darkMode.toggle();
-      }
-    });
-  }
-  if (darkMode && darkMode.isActivated()) {
-    darkMode.toggle();
+  const darkMode = store.get('darkmodeOption');
```

Comment:
Minor, but would be nice to keep the camel-case consistent and say "darkModeOption"